lib/prefab/client.rb

```diff
@@ -42,6 +42,11 @@ def initialize(options = Prefab::Options.new)
       end
     end
 
+    def set_thread_log_context(lookup_key, properties)
```

Comment:
I think a `.clear` method is probably a good idea too. 

Default usage should probably be as a controller `around` filter that cleans things up afterwards. Otherwise they'll live on that thread as it goes on to other things.

Reply:
I'm concerned about leaving them as well. What i did in java was accept a runnable/callable to run inside my context, and clear it when completed - see https://github.com/prefab-cloud/prefab-cloud-java/blob/main/client/src/main/java/cloud/prefab/client/config/logging/TargetedLoggingHelper.java#L20

Reply:
These are good concerns. I think I'll remove this method and instead expose a method that takes a block and resets in an `ensure`. 

That usage would require consumers to use an `around_action` or manually wrap code in a block.

Reply:
Updated!